csrc/transformer/general_kernels.cu

```diff
@@ -161,7 +161,7 @@ void launch_fused_add2<float>(float* out,
     int total_count = batch_size * seq_length * hidden_dim / 4;
     dim3 grid_dim = DS_GET_BLOCKS(total_count);  //(batch_size * seq_length);
 
-    dim3 block_dim = DS_CUDA_NUM_THREADS;  //(hidden_dim / 4);
+    dim3 block_dim = DS_CUDA_NUM_THREADS;        //(hidden_dim / 4);
```

Comment:
This is unintended, correct?

Reply:
These are just added by applying the pre-commit checks: `pre-commit run --all-files`

Maybe I'm using a different version on a `pre-commit` dependency?